.gitpod.yml

```diff
@@ -2,7 +2,31 @@ image:
   file: .gitpod.Dockerfile
 
 tasks:
-  - init: |
+  - name: Futurenet
+    init: |
+      docker pull stellar/quickstart:soroban-dev
+    command: |
+      docker run --rm -it \
+      --name stellar \
+      --platform linux/amd64 \
```

Comment:
`stellar/quickstart:soroban-dev` is now published with multi-arch support, so, I don't think you need to force the amd64 platform here, should be able to remove it.

Reply:
I removed the platform designation; Thank you.